r/R/examples/stat_large.R

```diff
@@ -0,0 +1,36 @@
+#source("../core.R")
+#source("../ui.R")
+library(h2owave)
+library(tokenizers)
+base_url <- "https://programminghistorian.org/assets/basic-text-processing-in-r"
+url <- sprintf("%s/sotu_text/236.txt", base_url)
+text <- paste(readLines(url), collapse = "\n")
+sentences <- tokenize_sentences(text)
+captions <- sentences[[1]][sample(seq(1,length(sentences[[1]])),1)]
```

Comment:
It would be good to hard-code the data right here (or use one of R's built-in datasets) - makes the example easier to read. In general, the shorter the example, the better.

Reply:
Updated with a standard text list.